core/trino-main/src/main/java/io/trino/execution/QueryManagerStats.java

```diff
@@ -272,4 +278,50 @@ public DistributionStat getCpuInputByteRate()
     {
         return cpuInputByteRate;
     }
+
+    @Managed
```

Comment:
I think you already have these in `TimeSharingTaskExecutor`, e.g: `io.trino.execution.executor.timesharing.TimeSharingTaskExecutor#getRunningSplits`,
`io.trino.execution.executor.timesharing.TimeSharingTaskExecutor#getWaitingSplits`
Probably just no `completedDrivers`, but I'm not sure it's important

Reply:
`TimeSharingTaskExecutor` is on worker - right? This is coordinator metric